vendor/hybridauth/Provider/Twitter.php

```diff
@@ -118,6 +122,8 @@ public function getUserProfile()
             'follows' => $data->get('friends_count'),
         ];
 
+		pre( $userProfile ); die;
```

Comment:
@sachyya Is this required or it was just for debugging and needs to be removed?

Reply:
This is just for debugging